core/src/main/java/bio/terra/pearl/core/model/audit/ResponsibleEntity.java

```diff
@@ -34,6 +35,12 @@ public ResponsibleEntity(String systemProcess) {
         }
         this.systemProcess = systemProcess;
     }
+    public ResponsibleEntity(PortalParticipantUser ppUser) {
+        if (ppUser == null) {
+            throw new IllegalArgumentException("portal participant user cannot be null");
+        }
+        this.participantUser = ppUser.getParticipantUser();
+    }
```

Comment:
I'm not 100% sure that doing this is correct, but I saw a few places where we take a PortalParticipantUser and set the responsibleEntity to the underlying ParticipantUser